### DISCUSSION TITLE
Suspicious activity in my GitActions

### DISCUSSION THREAD

**Discussion by RichardTMiles** [community/community#26598] [Actions]
<p>I have this run I noticed this morning in my actions that appears like a system update, but I’m afraid it may be doing things with crypto.</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/RichardTMiles/CarbonPHP/actions/runs/719332248" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:400/400;"><img src="https://avatars.githubusercontent.com/u/9538357?s=400&amp;amp;v=4" class="thumbnail" width="400" height="400"></div>

<h3><a href="https://github.com/RichardTMiles/CarbonPHP/actions/runs/719332248" target="_blank" rel="noopener nofollow ugc">RichardTMiles/CarbonPHP</a></h3>

<p><a href="https://github.com/RichardTMiles/CarbonPHP/actions/runs/719332248" target="_blank" rel="noopener nofollow ugc">ns/runs/719332248</a></p>

  <p><span class="label1">A PHP 7.4+ tool kit designed for SEO optimization </span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>The run above has nothing to do with my repository and when I click any link to the user or commit it 404’s. I know how containers work and as an open source project I don’t very much care about access if any, as I would assume not, in case of bad actors. I figure it maybe a bug, but best reported somewhere.</p>
<p>Best,<br>
Richard Miles</p>

**Comment by airtower-luna** [comment]
<p>There are some cryptomining attacks going on at the moment, so it might be one of those:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/5814919/181098621-9be72564-cf6d-4623-9bdc-01cee50404b5.jpeg" class="site-icon" width="150" height="150">
      <a href="https://therecord.media/github-investigating-crypto-mining-campaign-abusing-its-server-infrastructure/" target="_blank" rel="noopener" title="11:24PM - 02 April 2021">The Record by Recorded Future – 2 Apr 21</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/414;"><img src="https://user-images.githubusercontent.com/5814919/181098622-d184b4da-c707-44e3-b910-b1a335a56373.png" class="thumbnail" width="690" height="414" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/3/738534fdb794766ab378ef56bc5e3e0084db9fa4_2_690x414.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/3/738534fdb794766ab378ef56bc5e3e0084db9fa4_2_1035x621.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/7/3/738534fdb794766ab378ef56bc5e3e0084db9fa4_2_1380x828.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/3/738534fdb794766ab378ef56bc5e3e0084db9fa4_2_10x10.png"></div>

<h3><a href="https://therecord.media/github-investigating-crypto-mining-campaign-abusing-its-server-infrastructure/" target="_blank" rel="noopener">GitHub investigating crypto-mining campaign abusing its server infrastructure...</a></h3>

<p>Code-hosting service GitHub is actively investigating a series of attacks against its cloud infrastructure that allowed cybercriminals to implant and abuse the company's servers for illicit crypto-mining operations, a spokesperson told The Record...</p>

  <p><span class="label1">Est. reading time: 3 minutes</span>
    </p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>On the plus side I guess the 404 on user and pull request indicate that account has been blocked already.</p>

**Comment by ghost** [comment, marked as answer]
<p>There’s no permanent setting to limit contributions to only collaborators but the temporary interaction limits could be helpful in preventing this in the future.</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/communities/moderating-comments-and-conversations/limiting-interactions-in-your-repository" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/communities/moderating-comments-and-conversations/limiting-interactions-in-your-repository" target="_blank" rel="noopener">Limiting interactions in your repository - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/communities/moderating-comments-and-conversations/limiting-interactions-in-your-repository" target="_blank" rel="noopener">//docs.github.com/en/communities/moderating-comments-and-conversations/limiting-interactions-in-your-repository</a></p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>So you can enable them for up to 6 months but would have to enable again after that time period.</p>
<p>We have received feedback from other GitHub users that having a permanent interaction limit would be useful and we can definitely advocate for this feature to the Actions product team.</p>

**Reply by Lindaluburtron7** [reply]
I would like to view and if my home is advertised  as on market or applications inas for me to be controlled going in specific direction and documentation any negative reports about me in wrong information I would like to view and delete if necessary

**Comment by RichardTMiles** [comment]
<p>I wouldn’t care if it was related at all. It looks like spam. I can’t access the users account or the commit in question. I wouldn’t want bots using actions servers to mine crypto.</p>

**Comment by RichardTMiles** [comment]
<p>Glad everyone is aware :slight_smile:</p>

**Comment by dcantele** [comment]
<p>Thanks for the update - hope all goes well!!</p>
